GitHub.Collectors/Model/DefaultCollector.cs

```diff
@@ -49,56 +49,55 @@ public class DefaultCollector : ICollector
         {
-            // ToDo: kivancm: 08/09/2021, due to higher rate of 'secondary' rate limiting 429s from GitHub.com, disable the following to reduce the overall HTTP request against GitHub.com.
-            //JToken organizationUrlToken = jsonObject.SelectToken($"$.organization.url");
-            //if (organizationUrlToken != null)
-            //{
-            //    string organizationUrl = organizationUrlToken.Value<string>();
-            //    await this.ProcessUrlAsync(organizationUrl, OrganizationInstanceRecordType, allowlistedResponses: new List<HttpResponseSignature>()).ConfigureAwait(false);
-            //}
+            JToken organizationUrlToken = jsonObject.SelectToken($"$.organization.url");
+            if (organizationUrlToken != null)
+            {
+                string organizationUrl = organizationUrlToken.Value<string>();
+                await this.OffloadToPointCollector(organizationUrl, DataContract.OrganizationInstanceRecordType, DataContract.OrganizationsApiName, repository, "Object").ConfigureAwait(false);
+            }
+            
 
             JToken senderUrlToken = jsonObject.SelectToken($"$.sender.url");
             if (senderUrlToken != null)
             {
                 string senderUrl = senderUrlToken.Value<string>();
-                // There are cases where we receive this payload for removed team members, members no longer exist in GitHub. Therefore, the following call can fail with 404 not found.
-                // When using a GitHub app, the same request fails with 403: "Resource not accessible by integration" response.
-                List<HttpResponseSignature> allowlistedResponses = new List<HttpResponseSignature>()
-                {
-                    UserNotFoundResponse,
-                    ResourceNotAccessibleByIntegrationResponse,
-                };
-                await this.ProcessUrlAsync(senderUrl, UserInstanceRecordType, allowlistedResponses).ConfigureAwait(false);
+                await this.OffloadToPointCollector(senderUrl, DataContract.UserInstanceRecordType, DataContract.UsersApiName, repository, "Object").ConfigureAwait(false);
             }
         }
 
-        private async Task ProcessUrlAsync(string requestUrl, string recordType, List<HttpResponseSignature> allowlistedResponses)
+        private async Task OffloadToPointCollector(string url, string recordType, string apiName, Repository repository, string responseType = "Array")
         {
-            HttpResponseMessage response = await this.HttpClient.GetConditionalViaETagAsync(requestUrl, recordType, this.Authentication, allowlistedResponses).ConfigureAwait(false);
-            if (response.StatusCode == HttpStatusCode.NotModified)
+            ICache<PointCollectorTableEntity> pointCache = new AzureTableCache<PointCollectorTableEntity>(this.TelemetryClient, "point");
+            await pointCache.InitializeAsync().ConfigureAwait(false);
```

Comment:
Similar to other caches, I would initialize this cache at higher-level e.g., functions class and then pass to Webhook processor and then DefaultCollector. This way, you don't keep initializing it (even if that is optimized for that).